### PR TITLE
Code quality/add precommit

### PR DESCRIPTION
**Pull request by KPostOffice**
Add pre-commit to messaging module

### CODE REVIEW

File: thoth/messaging/__init__.py [hunk 1]
```diff
@@ -34,6 +34,17 @@
     UnresolvedPackageMessage,
 ]
 
+__all__ = [
```
Comment (pacospace) [inline thoth/messaging/__init__.py:37]:
```
__all__ = ALL_MESSAGES + ["MessageBase", "message_factory"]
```

Comment (KPostOffice) [reply thoth/messaging/__init__.py:37]: `ALL_MESSAGES` contains class references and `__all__` contains strings

Comment (pacospace) [reply thoth/messaging/__init__.py:37]: sorry, my bad :/

Comment (fridex) [reply thoth/messaging/__init__.py:39]:
```python
[msg_cls.__name__ for msg_cls in ALL_MESSAGES]
```

File: thoth/messaging/__init__.py [hunk 2]
```diff
@@ -36,6 +36,17 @@
...
+    "MessageBase",
+    "HashMismatchMessage",
+    "MissingPackageMessage",
+    "MissingVersionMessage",
+    "PackageReleaseMessage",
```
Comment (pacospace) [inline thoth/messaging/__init__.py:44]: SolvedPackageMessage?

**Pull request merged**